packages/styles/src/theme.ts

```diff
@@ -59,6 +59,10 @@ const componentsSpec: DeepString<HvThemeComponents> = {
     secondLevelItemColor: "string",
     secondLevelSelectedItemColor: "string",
     secondLevelSelectedItemBackgroundColor: "string",
+    selectedItemBorderRadius: "string",
+    secondLevelSelectedItemBorderRadius: "string",
+    itemPadding: "string",
+    secondLevelItemPadding: "string",
```

Comment:
It this API something we want to keep extending? 🤔

Reply:
Not really 😅 I only added this because we had some styling on the header component that only seemed to have been added for white labelling purposes (to get something like we have [here](https://lumada-design.github.io/uikit/master/?path=/docs/guides-theming-white-labeling--page)). 
Instead of removing the styling that was unnecessary I decided to add these tokens in case this is something we want to continue supporting. 
However, this is definitely something we need to look at when we review the theme.